### DISCUSSION TITLE
Deploy only changed files from a repository

### DISCUSSION THREAD

**Discussion by grahamreach** [deployphp/deployer#3340] [General]
Hi,

Apologies if this has been posted before, I had a look through the discussions but I couldn't find anything related.

As the title says, is it possible to deploy files from a git repository but select only the modified files? I ask because at the minute my deployments are trying to send every file which is obviously not ideal.

If so please could I be pointed in the right direction on how to resolve this?

Thanks,

Graham

**Comment by antonmedv** [comment]
> deployments are trying to send every file which is obviously not ideal.

this is not what Deployer does by default. Only the changes are fetched via git.

**Reply by grahamreach** [reply]
Hi @antonmedv,

Sorry I meant prior to using deployer it was trying to do every file. If that's deployers behaviour by default then that'll be perfect thanks. Just working through the docs to get it working
